crypto/bn/bn_lib.c

```diff
@@ -322,15 +322,21 @@ BIGNUM *BN_dup(const BIGNUM *a)
 
 BIGNUM *BN_copy(BIGNUM *a, const BIGNUM *b)
 {
+    int bn_words;
+
     bn_check_top(b);
 
+    bn_words = b->top;
+    if (BN_get_flags(b, BN_FLG_CONSTTIME))
+        bn_words = b->dmax;
```

Comment:
Just a fly-by suggestion: wouldn't it be more expressive to write the following instead?

```
    bn_words = BN_get_flags(b, BN_FLG_CONSTTIME) ? b->dmax : b->top;
```

Reply:
Good point! Thanks!